src/core/Renderer.js

```diff
@@ -371,9 +368,9 @@ export class Renderer {
             }
 
             // Same for stencil
-            if(this.stencil || (!target || target.stencil)) {
+            if (this.stencil || !target || target.stencil) {
```

Comment:
@DougLilliequist is this supposed to have an and (`&&`) operator like the depth buffer?
```js
if (this.stencil && (!target || target.stencil)) {
```

Reply:
It seems like it got merged broken since
```
this.stencil || !target || target.stencil
```
will be true if the target is null no matter what the stencil setting is...

Reply:
I don't think it's currently causing any errors, though that's why I asked.

@gordonnl we should probably fix that, even if it's not causing errors, the stencil mask is likely being set even when there is no stencil.